.buildkite/scripts/bootstrap.sh

```diff
@@ -26,6 +26,8 @@ fi
 ### upload ts-refs-cache artifacts as quickly as possible so they are available for download
 ###
 if [[ "${BUILD_TS_REFS_CACHE_CAPTURE:-}" == "true" ]]; then
+  echo "--- Build ts-refs-cache"
+  node scripts/build_ts_refs.js --ignore-type-failures
```

Comment:
Is it possible to move this to only run before `Upload ts-refs-cache` as to not extend the CI time?